src/library/scala/collection/BitSetLike.scala

```diff
@@ -39,11 +41,13 @@ trait BitSetLike[+This <: BitSetLike[This] with SortedSet[Int]] extends SortedSe
 
   /** The number of words (each with 64 bits) making up the set */
   protected def nwords: Int
+  @inline private[collection] final def _nwords = nwords
 
   /** The words at index `idx`, or 0L if outside the range of the set
    *  '''Note:''' requires `idx >= 0`
    */
   protected def word(idx: Int): Long
+  @inline private[collection] final def _word(idx: Int) = word(idx)
```

Comment:
what's the purpose of these two inline methods that just forward directly to other methods?

Reply:
Its to provide access to the properties where protected doesn't allow - fromthe other bitset in mutable.bitset

Reply:
unless I'm missing something, `mutable.BitSet` ought to be able to invoke protected members of `collection.BitSetLike`, since it inherits from it

Reply:
Yes -I thought so too. I didn't add it until it didn't compile though
There is some rework around here as I an optimising the flow, and it will be easy to rework when the code stabilises